src/pages/Post.js

```diff
@@ -559,6 +550,22 @@ class PostPage extends React.Component {
   render() {
     const { crop, croppedImageUrl, src } = this.state;
 
+    if (this.state.filters['toggle']['enabled']) {
+      this.showFilterBoxes = "block"
+      this.filterToggleText = "Remove Filters";
+      this.filterToggleColor = "#a32512";
+    } else {
+      this.showFilterBoxes = "none"
+      this.filterToggleText = "Add Filters";
+      this.filterToggleColor = "#12a340";
+    }
+
+    if (this.state.imageUrl !== null) {
+      this.showCropButton = "block"
+    } else {
+      this.showCropButton = "none"
+    }
```

Comment:
This would be cleaner using a ternary operator. For example, for the toggle text, you can rewrite it as:
```const filterToggleText = this.state.filtersEnabled ? "Remove Filters" : "Add Filters"```. This also removes the need for an unnecessary class variable.